.github/workflows/typescript_test.yml

```
@@ -342,10 +342,12 @@ jobs:
         shell: bash
         run: |
           touch .env
-          echo "${{ secrets.ENV_VARS }}" > .env
+          echo "${ENV_VARS}" > .env
 
+        env:
+          ENV_VARS: ${{ secrets.ENV_VARS }}
```

actionlint (1.7.11)

[error] property "env_vars" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; anthropic_api_key: string; github_token: string; openai_api_key: string; store_api_key: string; tavily_api_key: string}

(expression)